.gitlab-ci.yml

```diff
@@ -0,0 +1,15 @@
+image: ubuntu:bionic
+
+variables:
+  GRADLE_VERSION: "4.10.3"
+
+before_script:
+  - apt-get update && apt-get -y upgrade
```

Comment:
This line could be split into two lines for easier reading.

Reply:
OK.